pkg/artifacts/signable.go

```diff
@@ -336,9 +342,10 @@ func ExtractStructuredTargetFromResults(obj objects.TektonObject, categoryMarker
 		if strings.HasSuffix(res.Name, categoryMarker) {
 			valid, err := isStructuredResult(res, categoryMarker)
 			if err != nil {
-				logger.Debugf("ExtractStructuredTargetFromResults: %v", err)
```

Comment:
I don't think we want to error here - I think it's fine if a result happens to have the suffix but doesn't conform to the structured result format.

Reply:
+1 agree that we should not error here.
Currently we are just logging a message (although at debug level which does not show up in the logs which is not useful and as such would prefer WARN level atleast)  and continuing.